source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -73,19 +262,20 @@ following methods:
      - Description
 
    * - ``SetBypassDocumentValidation()`` 
-     - | Whether to allow the write to opt-out of document level validation.
+     - | Specifies whether the operation can opt-out of document level validation.
        | Default: ``false``
 
    * - ``SetOrdered()``
-     - | Whether to stop performing write operations after an error occurs. 
+     - | Specifies whether the driver stops performing write operations after an error occurs. 
        | Default: ``true``
 
-Return Values
-~~~~~~~~~~~~~
+Collection-Level Return Value
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The ``BulkWrite()`` method returns a ``BulkWriteResult`` type, which
-contains information about the bulk operation if it's successful. The
-``BulkWriteResult`` type contains the following properties:
+includes information about the bulk operation.
+
+The ``BulkWriteResult`` type contains the following properties:
```

Comment:
Do we need to add the `Acknowledged` property?

Reply:
added!